dvc/commands/add.py

```diff
@@ -140,6 +141,14 @@ def add_parser(subparsers, parent_parser):
         ),
         metavar="<number>",
     )
+    parser.add_argument(
+        "-f",
+        "--force",
+        action="store_true",
+        default=False,
+        help="Ignores overwrites to dvc.yaml file and allows overwriting local stage"
```

Comment:
What do you mean by `overwrites to dvc.yaml` here? When does `dvc add` affect `dvc.yaml`?

Reply:
ugh, my bad.
tl;dr - you're right and it doesn't. I'm changing this help message to be the same as others

what threw me off here is that add command actually creates single-stages for all targets and uses the stage objects to check the graph (🤷‍♂️ I guess shoving the targets into stages is a way to conform to updating the index which happens there).
So, when creating the stage objects, `force` is actually passed there, BUT, upon re-checking this logic, it's ineffectual in this flow (because it's only inspected if stage is created with `validate=True` to opt out of some validation 😄 
So, to summarize - I was mistaken, the force flag ONLY has the effect added in this PR to override local files/folders if `--out <PATH>` is specified

The end